### PR TITLE
Added routes, middleware and connection to db using es6

### PR DESCRIPTION
**Pull request by renecardozo**

### CODE REVIEW

**Comment by renecardozo** [comment]
@ArielYanarico hello please review this changes

File: ah-server/src/index.js [hunk 1]
```diff
@@ -1,19 +1,13 @@
...
-})
-
-app.listen(3000, () => {
+import bodyParser from 'body-parser';
+import Config from './config/config'
+import Db from './database/db-manager';
+const app = express();
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
+var routes = require('./routes/accouting-home.route');
```
Comment (ArielYanarico) [inline ah-server/src/index.js]: Maybe we could use import instead require here

**Comment by renecardozo** [comment]
@ArielYanarico it is done, please take a look

File: ah-server/src/database/db-manager.js [hunk 2]
```diff
@@ -1,19 +1,51 @@
-"use strict";
-
-// Food is a base class
-class Food {
-
-    constructor (name) {
-        this.name = name;
+import Db from './connection'
+const ObjectId = require('mongodb').ObjectID;
+let db = new Db();
```
Comment (ArielYanarico) [inline ah-server/src/database/db-manager.js]: You could put this variable into the DbManager class

**Comment by ArielYanarico** [comment]
@renecardozo  Looks Good

**Pull request merged**